msvc-full-features/vcpkg.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "cdda-vcpkg-dependencies",
-    "version-string": "0.F",
```

Comment:
Fwiw I don't think these feed into anything, but it's just a metadata key for the vcpkg.json file.

Reply:
Yeah they don't seem to have changed anything having been missed this long